arangod/IResearch/IResearchViewNode.cpp

```diff
@@ -174,6 +131,166 @@ std::vector<arangodb::iresearch::IResearchSort> fromVelocyPack(
   return sorts;
 }
 
+// -----------------------------------------------------------------------------
+// --SECTION--                            helpers for IResearchViewNode::Options
+// -----------------------------------------------------------------------------
+
+void toVelocyPack(
+    velocypack::Builder& builder,
+    arangodb::iresearch::IResearchViewNode::Options const& options
+) {
+  VPackObjectBuilder objectScope(&builder);
+  builder.add("forceSync", VPackValue(options.forceSync));
```

Comment:
can we rename `forceSync` to `waitForSync`, because we use `waitForSync` in all other places in ArangoDB.
Renaming it here would make it more consistent.